package/MDAnalysis/analysis/rms.py

```diff
@@ -167,21 +167,38 @@ def rmsd(a, b, weights=None, center=False, superposition=False):
 
     Parameters
     ----------
-    a, b : array_like
-        coordinates to align
+    a : array_like
+        coordinates to align to `b`
+    b : array_like
+        coordinates to align (matching `a`)
```

Comment:
https://github.com/MDAnalysis/mdanalysis/pull/1581#discussion_r132369096

The backtics are [standard with numpy style](https://github.com/numpy/numpy/blob/master/doc/example.py). I think @kain88-de also mentioned the inconsistency in the docs with asterisk/backtics but [our doc style guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#writing-docstrings) does not really prescribe anything.

I updated our example with backtics as in numpy. Feel free to start a discussion :-).